File: 1848. Minimum Distance to the Target Element/getMinDistance.py
class Solution:
    def getMinDistance(self, nums: List[int], target: int, start: int) -> int:
        minAbs=float('inf')
        for i in range(len(nums)):
            if(nums[i]==target):
               minAbs= min(minAbs, abs(i-start))

        return minAbs

File: 1848. Minimum Distance to the Target Element/getMinDistance.java
class Solution {
    public int getMinDistance(int[] nums, int target, int start) {
        int minAbs= Integer.MAX_VALUE;
        for(int i=0; i<nums.length; i++){
            if(nums[i]==target){
                int out=Math.abs(i-start);
                minAbs=Math.min(minAbs, out);
            }
        }
        return minAbs;
    }
}
